jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/PullAndCacheBaseImageLayerStep.java

```diff
@@ -82,6 +82,9 @@ public CachedLayer call() throws IOException, CacheCorruptedException {
       Optional<CachedLayer> optionalCachedLayer = cache.retrieve(layerDigest);
       if (optionalCachedLayer.isPresent()) {
         return optionalCachedLayer.get();
+      } else if (buildConfiguration.getOffline()) {
+        throw new IOException(
+            "Cannot run Jib in offline mode; Cache is missing image layer " + layerDigest);
```

Comment:
Chances are pretty low, but I can see this can really happen in practice. For example, Jib downloaded and saved the manifest, but while Jib downloading layers, you press Ctrl+C (haven't tried though), and you happen to run it in the offline mode next. This isn't exactly cache corruption (although in some sense yes). Let's be a little bit more specific, like "... local Jib cache for base images is missing ..." Maybe a bit more helpful to suggest running online once more. The error message is relevant to jib-core users too.